scalyr_agent/tests/smoke_tests/rpm_smoke_test.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+
+import os
+
+import pytest
+import docker
+import six
+
+from scalyr_agent.tests.smoke_tests.tools.utils import create_temp_dir_with_constant_name
+
+from .tools.package.base_builder import Builder
+from .tools.package.distribution_package_builder.build_image import DistributionPackageBuilder
+from .tools.package.rpm.build_image import RpmBaseDistributionImageBuilder
+from .tools.utils import copy_agent_source
+
+dockerfile = \
```

Comment:
Minor style / personal preference things - I prefer to store such larger template values in fixture files on disk instead of in-line.

Reply:
Yeah, It looks more readable. Done.